runtime/src/ecalls.rs

```diff
@@ -58,6 +58,20 @@ mod riscv32 {
         ecall!(3, b.as_ptr(), b.len(), _out);
     }
 
+    /// Bench cycles with input is function name
+    pub fn bench_cycles_ecall(s: &str) {
+        let mut _out: u32;
+        ecall!(4, s.as_ptr(), s.len(), _out);
+    }
+    /// An empty type representing the cycles
+    pub struct NexusBench;
+    impl core::fmt::Write for NexusBench {
+        fn write_str(&mut self, s: &str) -> Result<(), core::fmt::Error> {
+            bench_cycles_ecall(s);
+            Ok(())
+        }
+    }
```

Comment:
Why are the macro and `Write` implementation needed?

Reply:
It's not needed. I removed it in [6dd69d0](https://github.com/nexus-xyz/nexus-zkvm/pull/243/commits/6dd69d0f13e127f638cd2f061f65cf167686c5e9)